Adds admin commands to handle damaged island db

tastybento opened the pull request:
Trash, purge, etc.

DO NOT MERGE YET

https://github.com/BentoBoxWorld/BentoBox/issues/505

tastybento committed:
WIP - adds admin commands to handle damaged island db

Trash, purge, etc.

DO NOT MERGE YET

https://github.com/BentoBoxWorld/BentoBox/issues/505

Poslovitch reviewed (approved):
PR looks good! Give me 5 minutes and I'll merge it.

EDIT: do not worry, that's just a joke ;)

tastybento committed:
Prefixes the island uniqueId with the game mode name.

This enables manual removal of old game mode island files from the
database by admins if required.

tastybento committed:
Adds Admin trash and empty trash commands

tastybento committed:
Adds various commands for trash management

tastybento committed:
Merge remote-tracking branch 'origin/develop' into admintrash

tastybento committed:
Remove unused imports

Poslovitch committed:
Merge branch 'develop' into admintrash

Poslovitch committed:
Forgot the javadoc in IslandsManager
